source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -94,7 +94,8 @@ private CalificationsDto GetEvaluatorEvaluation(CalificationsEvaluationDto evalu
             {
                 View = UserView.Evaluation,
                 Evaluation = evaluationDto,
-                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                Califications = califications.Where
+                (c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)).ToList()
```

Comment:
I think this is the part I'm not understanding: 

> Will get the company evaluation: || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)

Evaluations are always from to a particular employee (evaluator / responsible) to another employee (evaluated).

As such:

> Will get his own: _loggedUser == c.EvaluatorEmployee

The current logged in user is the evaluator for this evaluation. This means that every evaluation that has this person as an evaluator / responsible will be brought.

> Will get the auto eval: || c.Owner == CalificationType.Auto

This is the case where the evaluation is an auto-evaluation. But where it the reference to the current user done?

> Will get the company evaluation: || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)

This is for evaluations that are ready for devolution and are owned by the company. But how do we know that the current user is related at all, as an evaluator or an evaluated person?